core-kotlin-modules/core-kotlin-10/src/test/kotlin/com/baeldung/variableshadowing/VariableShadowingUnitTest.kt

```diff
@@ -0,0 +1,177 @@
+@file:Suppress("UNUSED_VARIABLE", "UNUSED_ANONYMOUS_PARAMETER")
+
+package com.baeldung.variableshadowing
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class VariableShadowingUnitTest{
```

Comment:
Let's have separate tests instead of bunching them all together? You divided them at the moment using comment lines like:
// shadowing class member
// shadowing in loop
etc

These would be the test names instead